project_one/project1cmp464(4).html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <body>
+            <style>
+            .hidden{
+             font-family: 'Courier New', Courier, monospace;
+             color:black;
+             text-align: center;
+            
+            }
+            .a:hover{
+                        color:blue;
+                    }
+                    .a{
+                        color:black;
+                        position:absolute;
+                        bottom:10px;
+                       left:43%;
+                        
+                        
+                    }
+
+                    @media( max-width :600px) {
+    main{
+        width: 100%;
+    }
+        }
+            
+            </style>
+           
+                
+                <p  class="hidden"><b>This is section three & five</b></p>
+                <p><b>Click the button to see the message</b></p>
+
+                <button type="button" onclick="clickthebutton()" id="grading-element-three">Show A Thing</button>
+                <p id="grading-element-three-consequence"></p>
+                
+               <script>
+                  function clickthebutton(){
+                    document.getElementById("grading-element-three-consequence").innerHTML="Hey i was hidding here";
```

Comment:
very close to what i wanted but not quite. Can you have the text start inside the element, but have `grading-element-three-consequence` start as `visibility: hidden;` then after the click go to `visibility: visible`?

Reply:
document.querySelector('#grading-element-three-consequence').style.visibility
""
document.querySelector('#grading-element-three').click()
undefined
document.querySelector('#grading-element-three-consequence').style.visibility
"visible"
don't you ask for this? when you run it on console?